### PR TITLE
fix ltpa mbean for validation keys

### PR DESCRIPTION
**Pull request by Zech-Hein**
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.token.ltpa_fat

### CODE REVIEW

**Comment by Zech-Hein** [comment]
running another build for 2nd updated to line 404
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.token.ltpa_fat

File: dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/internal/LTPAConfigurationImpl.java [hunk 1]
```diff
@@ -289,15 +288,13 @@ private void resolveActualPrimaryKeysFileLocation() {
...
-                try {
-                    // primaryKeyImportFile has already been resolved when the server loads the config, this includes variable and .. being resolved.
-                    primaryKeyImportDir = new File(primaryKeyImportFile).getCanonicalFile().getParent() + File.separator;
-                    Tr.debug(tc, "primaryKeyImportDir: " + primaryKeyImportDir);
-                } catch (IOException e) {
-                    Tr.debug(tc, "An exception occurred in resolveActualPrimaryKeysFileLocation method", e);
-                }
+        if (monitorValidationKeysDir || isValidationKeysFileConfigured) {
+            try {
+                // primaryKeyImportFile has already been resolved when the server loads the config, this includes variable and .. being resolved.
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/internal/LTPAConfigurationImpl.java:293]:
```suggestion
                // primaryKeyImportFile has already been resolved when the server loads the config, this includes variable and .. being resolved.
                // primaryKeyImportDir is required to be set to load any validation keys.
```

**Comment by utle** [comment]
LGTM

**Review by utle — APPROVED** [review]

**Review by utle — COMMENTED** [review]
Have you test update trigger with polled and mbean with monitor validation keys dir set to false?
A) primary key
B) primary key and configure validationkeys

**Review by mshah0722 — APPROVED** [review]
LGTM!

**Comment by Zech-Hein** [comment]
#libby

**Pull request merged**